packages/next/test/fixtures/00-legacy-routes-nested/apps/app/package.json

```diff
@@ -3,8 +3,8 @@
     "build": "next build"
   },
   "dependencies": {
-    "next": "8.0.3",
```

Comment:
This should remain 8.0.3 since it;s testing legacy routes. Instead, pin engines to node 16.x